odl/space/fspace.py

```diff
@@ -503,6 +502,11 @@ def product(arg):
             return x_old(arg) * y_old(arg)
         y._function = product
 
+    def _inner(self, x1, x2):
```

Comment:
tbh simply let this pass through, it will fail in `LinearSpace._inner` anyway. This way causes more code at marginal gain.